etfl/core/genes.py

```diff
@@ -77,7 +78,27 @@ def transcribed_by(self):
     @transcribed_by.setter
     def transcribed_by(self,value):
         # TODO: Make this a setter that rewrites the adequate constraints
-        raise NotImplementedError()
+        if value != self._transcribed_by:
+            if self.model is None:
+                # Easy
+                self._transcribed_by = value
+            else:
+                # We need to make the model change the corresponding cstr
+                mod_id = self.id + '_transcription'
```

Comment:
Please replace by `get_transcription_reaction_name`

Reply:
You mean I should write a new function to do this?

Reply:
There is already a function for this [here](https://github.com/EPFL-LCSB/etfl/blob/master/etfl/core/memodel.py#L1672)